goal_src/jak1/pc/pckernel-common.gc

```diff
@@ -243,6 +248,18 @@
     (pc-sound-set-fade-hack fade-hack)
     )
 
+  (update-led obj)
```

Comment:
this seems like it will overwrite peoples LED to blue even if they don't have any of the options enabled?  Would be annoying if your controller's LED was not blue by default.

Reply:
well, like you said as soon as you set the led color you can't revert it back to "default" so I'm not sure that's gonna matter too much. Ideally we would have a "default color" setting somewhere.

Reply:
My point is that if a user has disabled all LED affecting settings, we should not still mess with their controller's LED -- as they may have set it externally / their controller has a different default color.  I'm not talking about us having our own default.

Reply:
DS4Windows seems to be the only program that sets the LED color and DS4Windows never sets it back to the color you selected unless you reconnect the controller (if it's yellow and you close the game, it will get stuck at yellow). I'm not sure if this is really worth it. I can add an arbitrary timer to prevent the LED from being set when the game boots if you have all settings off, but that's about it. Seems like a design flaw with the DS4 more than anything.

Reply:
Weirdly enough there seems to be some sort of mechanism that remembers what you set the LED to because if I set the color, close the game, reconnect my controller and open the game again it restores that last color.

Anyway, it seems like a fool's errand to handle not wanting to set the LED at all right now.